python/docs/events.md

```diff
@@ -616,20 +616,6 @@ Content for:
 - project channel:
   ```
   {
-      "workflowStatus": {... "workflow status object" ...}
-  }
-  ```
-
-
-#### `workflowstatuses.update`
```

Comment:
why delete this?

Reply:
Not intended, it's due to a bad rebase. It should be fixed now.